py-development.Rmd

```diff
@@ -428,6 +449,9 @@ all available function and intelligently filter them as you type out more
 letters, try it! When you start getting familiar with typing function names,
 you will notice that this is often faster than looking for functions in menus.
 
+The `import <package_name>` syntax is the most common and generic method of importing 
```

Comment:
Isn't this covered by the two bullets you added above or are you thinking about additional ways of importing?

Reply:
No, I think I added that first before the other section and didn't remember to remove it.  It can be deleted (I don't know how in this view).